src/adapters/manta.ts

```diff
@@ -60,6 +60,22 @@ export const calamariRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
       weightLimit: DEST_WEIGHT,
     },
   },
+  {
+    to: "kusama",
+    token: "KSM",
+    xcm: {
+      fee: { token: "KSM", amount: "11523248" },
+      weightLimit: DEST_WEIGHT,
+    },
+  },
+  {
+    to: "moonriver",
+    token: "MOVR",
```

Comment:
add a token: "KMA" represent: Calamari send KMA to Moonriver, and amount is: 10945110271985

Reply:
I will do this a bit later, sending KMA over XCM is not planned for initial release